irods/tests/update_docker_rs_image.yml

```diff
@@ -22,10 +22,14 @@
         - de-create-collection
         - delete-scheduled-rule
         - generateuuid
-        - rm-trash
         - set-uuid
         - sparcd-ingest
 
+    - name: remove rm-trash if present
+      file:
+        path: /var/lib/irods/msiExecCmd_bin/rm-trash
+        state: absent
+
```

Comment:
The docker image is built fresh each time, so you don't need to add removal logic to its playbook. Besides, this file holds unit tests.